src/App/App.js

```diff
@@ -0,0 +1,202 @@
+                    <button name="Family" onClick={(event) => this.handleDropdownValue(event)}>family</button>
+                    <button name="Fantasy" onClick={(event) => this.handleDropdownValue(event)}>fantasy</button>
+                    <button name="History" onClick={(event) => this.handleDropdownValue(event)}>history</button>
+                    <button name="Horror" onClick={(event) => this.handleDropdownValue(event)}>horror</button>
+                    <button name="Music" onClick={(event) => this.handleDropdownValue(event)}>music</button>
+                    <button name="Romance" onClick={(event) => this.handleDropdownValue(event)}>romance</button>
+                    <button name="Science fiction" onClick={(event) => this.handleDropdownValue(event)}>science fiction</button>
+                    <button name="Thriller" onClick={(event) => this.handleDropdownValue(event)}>thriller</button>
+                    <button name="War" onClick={(event) => this.handleDropdownValue(event)}>war</button>
+                    <button name="Western" onClick={(event) => this.handleDropdownValue(event)}>western</button>
+                  </DropdownButton>
+                  <p className="dropdown-text">with <span className="active-text">{this.state.ratingValue}</span> rating</p>
+                  <DropdownButton
+                    title=''
+                    className='rating-dropdown'>
+                    <button onClick={() => this.setState({ ratingValue: 'any' })}>any</button>
+                    <button name="one" value="1" onClick={(event) => this.handleRatingValue(event)}>one</button>
+                    <button name="two" value="2" onClick={(event) => this.handleRatingValue(event)}>two</button>
+                    <button name="three" value="3" onClick={(event) => this.handleRatingValue(event)}>three</button>
+                    <button name="four" value="4" onClick={(event) => this.handleRatingValue(event)}>four</button>
+                    <button name="five" value="5" onClick={(event) => this.handleRatingValue(event)}>five</button>
+                    <button name="six" value="6" onClick={(event) => this.handleRatingValue(event)}>six</button>
+                    <button name="seven" value="7" onClick={(event) => this.handleRatingValue(event)}>seven</button>
+                    <button name="eight" value="8" onClick={(event) => this.handleRatingValue(event)}>eight</button>
+                    <button name="nine" value="9" onClick={(event) => this.handleRatingValue(event)}>nine</button>
+                    <button name="ten" value="10" onClick={(event) => this.handleRatingValue(event)}>ten</button>
+                  </DropdownButton>
+                  {this.state.input !== '' || this.state.dropdownValue !== 'all' || this.state.ratingValue !== 'any' &&
+                    <button className="clear" onClick={() => this.clearFilters()}>clear</button>
```

Comment:
This could be:
`<button className="clear" onClick={this.clearFilters}>clear</button>`